### PR TITLE
Wrap Icon in a padded Box to reposition the tooltip.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/chat/conversation/messages/react-button/emoji-row/index.tsx [hunk 1]
```diff
@@ -13,6 +13,15 @@ type Props = {
   style?: Styles.StylesCrossPlatform
 }
 
+const HoverBox = Styles.isMobile
+  ? Kb.Box
+  : Styles.styled(Kb.Box)({
+      ...Styles.desktopStyles.clickable,
+      ':hover .icon': {
+        color: Styles.globalColors.blue,
```
Comment (buoyad) [inline shared/chat/conversation/messages/react-button/emoji-row/index.tsx]: we have [some classes](https://github.com/keybase/client/blob/master/shared/styles/index.desktop.tsx#L122) that are supposed to do this, try `hover_color_blue`

Comment (adamjspooner) [reply shared/chat/conversation/messages/react-button/emoji-row/index.tsx]: Sadly, that doesn't work in this case (`Icon` in a `Box2`).

Comment (chrisnojima) [reply shared/chat/conversation/messages/react-button/emoji-row/index.tsx]: use hover-container, see style.css

Comment (adamjspooner) [reply shared/chat/conversation/messages/react-button/emoji-row/index.tsx]: Thanks!

**Pull request merged**